documentation/index.md

```diff
@@ -0,0 +1,66 @@
+The Charmed Temporal K8s Operator delivers automated operations management from
+day 0 to day 2 on [Temporal](https://temporal.io/). It is an open source,
+end-to-end, production-ready workflow engine on top of [Juju](https://juju.is/).
+
+Temporal is a developer-first, open source platform that ensures the successful
+execution of services and applications (using workflows). Use Workflow as Code
+(TM) to build and operate resilient applications. Leverage developer friendly
+primitives and avoid fighting your infrastructure
+
+This operator provides a Temporal server, and consists of Python scripts which
+wraps the versions distributed by
+[temporalio](https://hub.docker.com/r/temporalio/server).
+
+The Charmed Temporal K8s operator offers features such as replication,
+observability and easy to use integration with applications. It addresses the
+requirement for deploying Temporal in a structured and uniform fashion, while
+allowing the user flexibility in configuration. It streamlines the process of
+deploying, scaling, configuring and overseeing Temporal at scale in a dependable
+manner for production purposes.
+
+## Project and community
```

Comment:
Other files use title case, so `Community` with a capital C? Same for other headers in this file.